### PR TITLE
provide missing `publish()` function overload for qt

### PR DESCRIPTION
**Pull request by Xavrax**
fix: Provide missing `publish()` function overload for QT wrapper

Provide missing `publish()` function overload for QT wrapper that allows set publish related options

### CODE REVIEW

File: qt/pubnub_qt.cpp [hunk 1]
```diff
@@ -530,6 +530,23 @@ pubnub_res pubnub_qt::publish(QString const& channel, QString const& message)
 }
 
 
+pubnub_res pubnub_qt::publish(QString const& channel, QString const& message, publish_options& opt)
+{
+    QMutexLocker lk(&d_mutex);
+    d_method = pubnubSendViaGET;
```
Comment (parfeon) [inline qt/pubnub_qt.cpp:536]: With provided way to set publish options, this feel redundant:
```suggestion
```

**Review by parfeon — CHANGES_REQUESTED** [review]
One small change required and the rest is good!

File: qt/pubnub_qt.cpp (hunk 1 above)
Comment (Xavrax) [reply qt/pubnub_qt.cpp]: You're right. I just copy-pasted the whole function on the beginning and forgot to adjust that. Thanks!

**Review by parfeon — APPROVED** [review]
LGTM!

**Comment by Xavrax** [comment]
@pubnub-release-bot release

**Pull request merged**